spec.bs

```diff
@@ -396,7 +399,7 @@ Each {{URLPattern}} object has an associated <dfn for=URLPattern>hash component<
 <div algorithm>
   To <dfn>canonicalize protocol</dfn> given a string |value| and a string |type|:
 
-  1. Let |strippedValue| be the given |value| with a single trailing U+003A (:) removed, if any.
+  1. Let |strippedValue| be the given |value| with a single trailing U+003A (`:`) removed, if any.
```

Comment:
Making these be in code font is not typical but it's a fine style choice if you find it cleaner :)